Assets/LeapMotion/Core/Scripts/Attributes/CombinablePropertyAttribute.cs

```diff
@@ -129,4 +130,43 @@ public abstract class CombinablePropertyAttribute : PropertyAttribute {
     public virtual void OnPropertyChanged(SerializedProperty property) { }
 #endif
   }
+
+  public static class AttributeExtensions {
+
+    public static MethodInfo FindMethodByName(this Type type,
```

Comment:
The class Type already has a method called GetMethod that takes a string argument.  What makes this method different?  I think that the name of the method should reflect what makes it special.

Reply:
Also, why is this extension method placed inside of the CombinablePropertyAttribute class?

Reply:
because it was copied from Quickbutton -- I still need to make Quickbutton use this method instead, so I'll do that. But this sort of reflection has been useful for attributes specifically, is why. I can move it to some.. reflection utility alternatively?